### ISSUE TITLE
cm-12.1 file systerm problem of the internal storage

### ISSUE THREAD

**Issue by zhou0** [MI3TD/android_device_xiaomi_pisces#40]
I have a Mi3-TD with 64gb storage but i can only use about half of the storage space.  
Will wiping  the internal storage fix this problem? or can this be a software problem?

**Comment by xuefer** [comment]
Use TWRP resize

**Comment by zhou0** [comment]
I run into a bigger problem now.   I drooped a parted binary into the system and use parted to removed partion 24,25,26,27 and 28 and set the system partition 24 (system ) and partition 25 (system1 ) to 1GB size。 partition 26 (cache ) to 512mb size , partition 27 （data ） is set to 8GB and partition 28 （internal sdcard）is reduced accordingly。     after doing this I flashed CM 12. During which an error message  shows up saying 掛載 /data 失敗。   after rebooting, the devices boots O.K. but ask for password， after i input the password it shows 解密失敗 and offers a button to factory reseting the device。 Tapping the button will cause the device to boot into recovery and start factory-resting automatically then reboot 。 after rebooting it goes to the same    解密失敗  error screen again.    

I though that flashing the stock tgz image would re-partition the emmc, but even flahsing the stock tgz image can not solve this problem.   

Is there a way to fix this?
